.travis.yml

```diff
@@ -37,7 +47,9 @@ install:
   - ./build/mvn --version
 
 script:
-  - ./build/mvn clean install -Dmaven.javadoc.skip=true -V
+  - export MVN_ARGS="-Dmaven.javadoc.skip=true -V -B -ntp -Dorg.slf4j.simpleLogger.defaultLogLevel=warn"
+  - ./build/mvn clean install -DskipTests $MVN_ARGS
+# - ./build/mvn test $MVN_ARGS
```

Comment:
Since we have some issues with zookeeper, we can change it to `build/mvn clean -pl kyuubi-common -am` and merge this PR, then fix other issues and enable those module's test later.

Reply:
+1 from me on this suggestion!